### PR TITLE
[Stablehlo] add conversion for AtenFlipOp

### PR DESCRIPTION
**Pull request by qingyunqu**

### CODE REVIEW

File: lib/Conversion/TorchToLinalg/Linear.cpp [hunk 1]
```diff
@@ -113,6 +113,10 @@ class ConvertAtenFlipOp : public OpConversionPattern<AtenFlipOp> {
     if (!matchPattern(adaptor.getDims(), m_TorchListOfConstantInts(axis)))
       return rewriter.notifyMatchFailure(op,
                                          "only constant dim lists supported");
+    for (unsigned i = 0, e = axis.size(); i < e; i++) {
+      axis[i] = toPositiveDim(axis[i], selfRank);
```
Comment (ramiro050) [inline lib/Conversion/TorchToLinalg/Linear.cpp:117]: Can you also add an `isValidDim` check like

https://github.com/llvm/torch-mlir/blob/0caaf8d32abc73e6dee6ff3b39573cc7fdc2929f/lib/Conversion/TorchToStablehlo/ViewLike.cpp#L360-L362

File: lib/Conversion/TorchToStablehlo/Basic.cpp [hunk 2]
```diff
@@ -1575,6 +1576,28 @@ LogicalResult ConvertAtenOp<AtenFillScalarOp>::matchAndRewrite(
...
+  Value self = adaptor.getSelf();
+  auto outType =
+      getTypeConverter()->convertType(op.getType()).cast<RankedTensorType>();
+
+  SmallVector<int64_t> dims;
+  if (!matchPattern(op.getDims(), m_TorchListOfConstantInts(dims))) {
+    return rewriter.notifyMatchFailure(op, "dims must be a list of const int");
+  }
+  for (unsigned i = 0, e = dims.size(); i < e; i++) {
+    dims[i] = toPositiveDim(dims[i], outType.getRank());
```
Comment (ramiro050) [inline lib/Conversion/TorchToStablehlo/Basic.cpp:1593]: Same comment as above

**Review by ramiro050 — APPROVED** [review]
LGTM, just a small change request

**Comment by qingyunqu** [comment]
@ashay Could you please help to resolve the CI failure? (Because of updating mlir-hlo.

**Comment by ashay** [comment]
@powderluv is there something specific that you do after the mhlo submodule is updated?  I tried restarting the build, but that didn't fix anything.

**Comment by qingyunqu** [comment]
> @powderluv is there something specific that you do after the mhlo submodule is updated? I tried restarting the build, but that didn't fix anything.

Seems that CI failed with same status after updating mlir-hlo everytime.

**Comment by powderluv** [comment]
Yeah I'm not sure what it is about that repo but hopefully soon we will be moving to stablehlo repo

**Pull request merged**